src/sentry/static/sentry/app/views/settings/settingsIndex.tsx

```diff
@@ -0,0 +1,388 @@
+
+            <GridPanel>
+              <HomePanelHeader>
+                <HomeLinkIcon to={LINKS.API}>
+                  <HomeIcon>
+                    <InlineSvg src="icon-lock" size="48px" />
+                  </HomeIcon>
+                  {t('API Keys')}
+                </HomeLinkIcon>
+              </HomePanelHeader>
+
+              <HomePanelBody>
+                <h3>{t('Quick links')}:</h3>
+                <ul>
+                  <li>
+                    <HomeLink to={LINKS.API}>{t('Auth Tokens')}</HomeLink>
+                  </li>
+                  <li>
+                    <HomeLink to={LINKS.API_APPLICATIONS}>{t('Applications')}</HomeLink>
+                  </li>
+                  <li>
+                    <ExternalHomeLink href={LINKS.DOCUMENTATION_API}>
+                      {t('Documentation')}
+                    </ExternalHomeLink>
+                  </li>
+                </ul>
+              </HomePanelBody>
+            </GridPanel>
+          </GridLayout>
+        </SettingsLayout>
+      </DocumentTitle>
+    );
+  }
+}
+
+export {SettingsIndex};
+export default withLatestContext(SettingsIndex);
+
+const HomePanelHeader = styled(PanelHeader)`
+  background: #fff;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  font-size: 18px;
+  text-transform: unset;
+  padding: 35px 30px;
+`;
+
+const HomePanelBody = styled(PanelBody)`
+  padding: 30px;
+
+  h3 {
+    font-size: 14px;
+  }
+
+  ul {
+    margin: 0;
+    li {
+      line-height: 1.6;
+      /* Bullet color */
+      color: ${p => p.theme.gray1};
+    }
+  }
+`;
+
+const getHomeIconMargin = css`
+  margin-bottom: 20px;
+`;
+
+const HomeIcon = styled('div')<{color?: string}>`
+  background: ${p => p.theme[p.color || 'gray2']};
+  color: #fff;
+  width: ${HOME_ICON_SIZE}px;
+  height: ${HOME_ICON_SIZE}px;
+  border-radius: ${HOME_ICON_SIZE}px;
+  ${getHomeIconMargin} > svg {
+    margin-top: 14px;
+  }
+`;
+
+type CenterableProps = {
+  isCentered?: boolean;
+};
+
+const HomeLink = styled(Link)`
+  color: ${p => p.theme.purple};
+
+  &:hover {
+    color: ${p => p.theme.purpleDark};
+  }
+`;
+
+const HomeLinkIcon = styled(HomeLink)`
+  overflow: hidden;
+  width: 100%;
+  ${flexCenter};
+`;
+
+const ExternalHomeLink = styled(
+  ({
+    isCentered,
+    ...props
+  }: CenterableProps & React.ComponentProps<typeof ExternalLink>) => (
+    <ExternalLink {...props} />
+  )
+)<CenterableProps>`
+  color: ${p => p.theme.purple};
+
+  &:hover {
+    color: ${p => p.theme.purpleDark};
+  }
+
+  ${p => p.isCentered && flexCenter};
+`;
+
+type SupportLinkProps = {
+  isOnPremise: boolean;
+  href: string;
+  to: string;
+  isCentered?: boolean;
+};
+
+const SupportLinkComponent = ({
+  isCentered,
+  isOnPremise,
+  href,
+  to,
+  ...props
+}: SupportLinkProps &
+  (
+    | {isOnPremise: true} & React.ComponentProps<typeof ExternalLink>
+    | {isOnPremise: false} & React.ComponentProps<typeof HomeLink>)) => {
```

Comment:
I might just slam these onto the `SupportLinkProps` type above, since we don't use that type anywhere else. Should make it easier to read